### PR TITLE
Fixed pull the lever command for inbetween words

### PR DESCRIPTION
**Pull request by NFMynster**

### CODE REVIEW

File: src/Runner.Discord/Responders/PullTheLeverResponder.cs [hunk 1]
```diff
@@ -74,24 +74,26 @@ private async Task<bool> CheckWithinRateLimit(IUser user)
 
         public async Task ProcessMessage(IMessage message, CancellationToken token)
         {
+            string storedMessage = message.Content.ToLower();
```
Comment (alanedwardes) [inline src/Runner.Discord/Responders/PullTheLeverResponder.cs:77]: `messageContent`? nothing is stored

Comment (NFMynster) [reply src/Runner.Discord/Responders/PullTheLeverResponder.cs:77]: Hating on my naming conventions? :P

Comment (NFMynster) [reply src/Runner.Discord/Responders/PullTheLeverResponder.cs:77]: Also this does not fix the command from being trigger from say, "lever the pull" but who cares, it's all in the bot channel

Comment (alanedwardes) [reply src/Runner.Discord/Responders/PullTheLeverResponder.cs:77]: that's fine - asking about the naming as reading this back doesn't make sense -- stored implies it was stored somewhere

Comment (NFMynster) [reply src/Runner.Discord/Responders/PullTheLeverResponder.cs:77]: Lemme clone again, this was a in-browser edit

**Pull request closed without merging**